app/helpers/checkout_helper.rb

```diff
@@ -55,7 +55,6 @@ def display_line_item_tax_rates(line_item)
 
   def display_adjustment_tax_rates(adjustment)
```

Comment:
Yeah, but it's in another class and we should really add as much documentation as possible IMHO. It really pays off, especially for open source projects like this.